lib/jsdom/living/svg/basic-types.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+// https://svgwg.org/svg2-draft/types.html#TermDetach
+function detach(value) {
+  switch (true) {
+    case typeof value === "string":
+      break;
+    default:
+      throw new TypeError(`jsdom internal error: detaching object of wrong type ${value}`);
+  }
+}
+exports.detach = detach;
+
+// https://svgwg.org/svg2-draft/types.html#TermAttach
+function attach(value, listObject) { // eslint-disable-line no-unused-vars
+  switch (true) {
```

Comment:
Why `switch(true)` instead of if-else? Is the unused variable for future use? Might want to add a brief comment to explain.

Reply:
+1, here and above not a fan of the switch(true) pattern.

Reply:
Will add comment for unused variable. `switch (true)` isn't needed now, but is there for parity with spec and will be helpful if/when the other basic SVG types are implemented.